cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -750,11 +750,13 @@ def _validate_colors_in_uns_dict(self, uns_dict: dict) -> None:
         for key, value in uns_dict.items():
             if key.endswith("_colors"):
                 # 1. Verify that the corresponding categorical field exists in obs
-                obs_unique_values = category_mapping.get(key.replace("_colors", ""))
+                column_name = key.replace("_colors", "")
+                obs_unique_values = category_mapping.get(column_name)
                 if not obs_unique_values:
-                    self.errors.append(
-                        f"Colors field uns[{key}] does not have a corresponding categorical field in obs"
-                    )
+                    error_message = f"Colors field uns[{key}] does not have a corresponding categorical field in obs"
```

Comment:
nit: add a period, for clarity in cases where the second error sentence is appended

Reply:
updated!